### ISSUE TITLE
Native Touchscreen Support

### ISSUE THREAD

**Issue by coolpeter98** [Quaver/Quaver#4283]
### Is your feature request related to a problem? Please describe.

I would really like to play this game on my touchscreen device (Steam Deck) however that is not possible due to the lack of native touchscreen support. Playing with the physical triggers feel like crap as they are very heavy and hard to actuate. This feature would benefit people using touchscreen devices to play.

### Describe the solution you'd like

I want something that looks like the following graphics I made:
1:
![image](https://github.com/user-attachments/assets/3885a657-3736-4241-b2b1-9eb359d76013)
2:
![image](https://github.com/user-attachments/assets/533d6d75-b5c4-4b9f-b9d7-63a0230a0961)

I think 2 is less in-the-way but 1 is pretty good too.

### Describe alternatives you've considered

I don't think there's any good alternatives to this suggestion.

**Comment by EpiCreeper** [comment]
From all the places ive seen this idea in funky friday. Cringe aside, option 1 is much better than 2 input wise because with the lack of physical feedback you get from a touch screen your fingers move around a lot, and also option 2 would cramp yours hands into each other. Id go with option 1 but with less visual distructions

**Comment by coolpeter98** [comment]
> From all the places ive seen this idea in funky friday. Cringe aside, option 1 is much better than 2 input wise because with the lack of physical feedback you get from a touch screen your fingers move around a lot, and also option 2 would cramp yours hands into each other. Id go with option 1 but with less visual distructions

True, Maybe disable the pressed button glow and just show the dividers.
